src/utils/createCoreModule/utils.ts

```diff
@@ -0,0 +1,48 @@
+import * as R from 'ramda';
+import { CoreModuleName } from '@/utils/createCoreModule';
+import { CorePath } from '@/utils/CoreAPI';
+import { StoreRow } from '@/store/types';
+
+type ObjectWithUUID<T extends object> = T & { uuid: string };
+
+export function hasUUID<T extends object>(obj: T): obj is ObjectWithUUID<T> {
+  return R.has('uuid', obj);
+}
+
+// TODO: Try using R
```

Comment:
need todo still?

Reply:
tbh I don't know what that is will investigate